oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -925,9 +980,24 @@ int mca_spml_ucx_put_nb_wprogress(shmem_ctx_t ctx, void* dst_addr, size_t size,
     mca_spml_ucx_ctx_t *ucx_ctx = (mca_spml_ucx_ctx_t *)ctx;
 
     ucx_mkey = mca_spml_ucx_get_mkey(ctx, dst, dst_addr, &rva, &mca_spml_ucx);
+#if HAVE_DECL_UCP_PUT_NBX
+    ucs_status_ptr_t status_ptr;
+    ucp_request_param_t param = {0};
```

Comment:
mixed decl/code
need to unite line 989 and line 979

Reply:
Not sure where I got what you meant here.

Reply:
can do
```
spml_ucx_mkey_t *ucx_mkey = mca_spml_ucx_get_mkey(ctx, dst, dst_addr, &rva, &mca_spml_ucx);
```
do convert the statement to declaration and avoid mix decl/code
(same in all places)

Reply:
Done, please let me know if that's what you meant.